### DISCUSSION TITLE
Just don't understand

### DISCUSSION THREAD

**Discussion by thepixy** [microsoft/terminal#15968] [Q&A] [answered]
I've used a lot of terminal programs in my lifetime. This one I found on the microsoft store and I just don't get it. There's nothing in my DOWNLOADS folder to install but I can oPEN it from within the MS store. It seems to open powershell. No instructions, no setup, not even typical selection of serial port, baud rate etc., and certainly no typical terminal settings. I saw a ling to "windows Terminal Support", and all it did was take me to Github. Glad I have an account here, but still nothing but a screen to write an issue?  So I guess I'll poke around some, but my only complaint is that this is nothing like what is expected by an ordinary user when it comes to downloading a program, trying to use it, and finding any obvious "how to" documentation. Pretty sure a lot of people who have used a terminal program don't have a Github account.

**Comment by bw1faeh0** [comment, marked as answer]
`Windows Terminal` is not a software to create a serial connection using RS232 or similar hardware.

It is a Terminal emulator. A terminal emulation is a computer program that emulates the function of a computer terminal. It is used to allow text-based programs to be used within a graphical user interface.

Inside Windows Terminal you can run e.g. text-based programs that connect to other hardware using RS232 and asking for settings like baudrate, partity, etc. I use Windows terminal to run following command: 
`plink -serial COM1 -sercfg 115200,8,1,N,N` to connect to my external hardware...

**Reply by thepixy** [reply]
I see. My rant was misdirected. It should have been directed at Microsoft, because it was one of several options on the Microsoft store after specifically searching for serial port terminal programs.  I had been trying several terminal programs is hopes of finding one that most closely emulated the "serial monitor"  that comes with all the Arduino IDE versions, so I thought I'd check the Microsoft store, and this program was the first one in their list. Guess I was hoping to find the old HyperTerm program, but I now realize that MS didn't really own it and could no longer distribute it. But anyway, thanks for the clarification.

**Reply by zadjii-msft** [reply]
That's correct. The Windows Terminal is not _specifically_ a serial port terminal application - it's a general-purpose terminal emulator. You can configure it with a profile to handle serial port connections, for sure. 

More dedicated serial connection support is tracked in #1280.

**Comment by maddes8cht** [comment]
> There's nothing in my DOWNLOADS folder to install but I can oPEN it from within the MS store

That is because it is already installed. Windows Terminal has become the default terminal host in Windows 10 /11 for a while now, and it may be installed by windows update.
However, strange thing: the version from the windows store and from winget are outdatet they still point to the 1.16 release.
You get the current release in `releases` in this repo.

> It seems to open powershell.

Yes, it opens a powershell inside the windows terminal.
Windows terminal beeing the surrounding application with the tabs-interface, its `Settings`, and it's overall ability to host a text-console like Powershell, cmd, bash or anything the like.

Windows Terminal is the default applikation in Windows to open Powershell or cmd and can host a bunch of other terminals  like bash too.

**Comment by zadjii-msft** [comment]
To add: You can find the docs for the Terminal linked on the About dialog, which should take you to https://aka.ms/terminal-documentation

**Reply by thepixy** [reply]
Thanks!
